### PR TITLE
*.: fix default mysql version 5.7.33 to 5.7.34

### PR DESCRIPTION
**Pull request by hustjieke**
### Which issue(s) this PR fixes?

Fixes #

### What this PR does?

Summary:

### CODE REVIEW

**Review by acekingke — COMMENTED** [review]
LGTM

File: utils/constants.go [hunk 1]
```diff
@@ -18,16 +18,16 @@ package utils
...
 	// MySQLTagsToSemVer maps simple version to semver versions
 	MySQLTagsToSemVer = map[string]string{
-		"5.7": "5.7.33",
+		"5.7": "5.7.34",
 	}
 
 	// MysqlImageVersions is a map of supported mysql version and their image
 	MysqlImageVersions = map[string]string{
-		"5.7.33": "percona/percona-server:5.7.33",
+		"5.7.34": "percona/percona-server:5.7.34",
```
Comment (zhyass) [inline utils/constants.go:30]: https://github.com/radondb/radondb-mysql-kubernetes/issues/200#issuecomment-912228878

Comment (hustjieke) [reply utils/constants.go:30]: good idea,  tks.

Comment (andyli029) [reply utils/constants.go:30]: Good advice.

Comment (andyli029) [reply utils/constants.go:30]: Good advice.

**Comment by hustjieke** [comment]
```
hustjason:radondb-mysql-kubernetes gaoriyao$ kubectl logs sample-mysql-0 -c init-mysql
Initializing database
2021-09-09T03:38:16.148708-00:00 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --
...
...
2021-09-09T03:38:26.887019-00:00 0 [Note] mysqld (mysqld 5.7.34-37-log) starting as process 73 ...
```

**Review by andyli029 — COMMENTED** [review]
LGTM.

**Pull request merged**